### ISSUE TITLE
Modular app.js

### ISSUE THREAD

**Issue by tatticoder** [RishabhDevbanshi/Pacman-Game#18]
All the code is currently in app.js
I think separating the code in **different files** and make it modular will make project more structured.
MVC separates an application into three components - **Model**, **View**, and **Controller**. 
This will make it easier for collaborators to work together and make merging requests easier by reducing conflicts.
Adding new updates and features will also be easier in future.

**Comment by RishabhDevbanshi** [comment]
Sounds cool ! Try to work on it , if you want any help feel free to contact.

**Comment by osoba** [comment]
Yeah I was going to suggest the same. The way the game is made right now is to check for CSS classes for any game logic, which is not ideal. It should definitely be separated into MVC components.

**Comment by pro-nav** [comment]
@RishabhDevbanshi Can I work on this? I think adding more levels will be great too.

**Comment by RishabhDevbanshi** [comment]
> @RishabhDevbanshi Can I work on this? I think adding more levels will be great too.

Sure !
